### DISCUSSION TITLE
libheif-dev: dependencies issue

### DISCUSSION THREAD

**Discussion by diramazioni** [linuxmint/discussions#214] [Questions]
I need to install libheif-dev (ros2 dependencies) but it says it depends on `libheif1 (= 1.12.0-2build1) but version 1.14.2-1ubuntu1 is installed`
It's a freshly installed Linux Mint 21.2 Victoria. 
What should I do, and where to file a bug?

**Comment by mtwebster** [comment]
Did run `apt update` and  retry?

Both should be at 1.14.2:

```
[07:26:43 PM] mtwebster@mintbox3:/usr/lib> apt policy libheif-dev
libheif-dev:
  Installed: 1.14.2-1ubuntu1
  Candidate: 1.14.2-1ubuntu1
  Version table:
 *** 1.14.2-1ubuntu1 500
        500 http://packages.linuxmint.com victoria/backport amd64 Packages
        100 /var/lib/dpkg/status
     1.12.0-2build1 500
        500 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages
[09:38:22 AM] mtwebster@mintbox3:/usr/lib> apt policy libheif1
libheif1:
  Installed: 1.14.2-1ubuntu1
  Candidate: 1.14.2-1ubuntu1
  Version table:
 *** 1.14.2-1ubuntu1 500
        500 http://packages.linuxmint.com victoria/backport amd64 Packages
        100 /var/lib/dpkg/status
     1.12.0-2build1 500
        500 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages
[09:38:39 AM] mtwebster@mintbox3:/usr/lib> 
```
Try installing both at once (even though the lib is installed):
```
apt install libheif1 libheif-dev
```

**Comment by diramazioni** [comment]
hello, yes, of course the packages were updated and can't install together. 

I solved by [manually downloading](https://mint.pkgs.org/21.2/mint-backport-amd64/libheif-dev_1.14.2-1ubuntu1_amd64.deb.html) and installing from mint backport. 
But why it's not in my source list? How to add mint-backport?

**Comment by mtwebster** [comment]
They should already be there in /etc/apt/sources.list.d/official -
```
deb http://packages.linuxmint.com victoria main upstream import backport
```

Also, that site you got the packages from is *not* official - they should not be labeling themselves as such. In the future you should use http://packages.linuxmint.com/ if you need to pick individual packages to download.
